app/Models/Post.php

```diff
@@ -49,11 +49,17 @@ public function signup()
     }
 
     /**
-     * Each post has many reactions.
+     * Query for total reactions for a post.
+     *
+     * @param int $postableId
+     * @param int $postableType
+     * @return int total count
      */
-    public function reactions()
+    public static function withReactionCount($postableId, $postableType)
     {
-        return $this->hasMany(Reaction::class);
+        return self::withCount(['reactions' => function ($query) use ($postableId, $postableType) {
```

Comment:
@DFurnes ok! I think this would be in the Photo Model though now? Is that right?